conf/application.conf

```diff
@@ -54,6 +54,7 @@ play.http.flash.httpOnly = true
 microservice {
 
   url = "http://localhost:9302"
+  iv-redirect = "/iv-uplift"
```

Comment:
Bookmarking this with a comment since depdening on the answer to the question in AppConfig, this might not be needed

Reply:
Correct. Not needed and removed.